src/components/__module.ts

```diff
@@ -37,6 +38,11 @@ export default class Module<T extends ModuleNodes = {}> {
    */
   protected config: EditorConfig;
 
+  /**
+   * Method names that must be decorated
+   */
+  protected decorateList: string[] = [];
```

Comment:
```suggestion
  protected methodsToDisableInReadonly: string[] = [];
```

Reply:
I think we should make another abstract module called `ApiModule extends Module`. All the API modules will extend the `ApiModule`. And the `methodsToDisableInReadonly` should be at `ApiModule` because it is not related to other editor modules.

Reply:
Yes, I also wanted to do this way. Thanks